rc/filetype/ocaml.kak

```diff
@@ -74,4 +74,22 @@ evaluate-commands %sh{
   "
 }
 
+# Conveniences
+# ‾‾‾‾‾‾‾‾‾‾‾‾
+
+# C has header and source files and you need to often switch between them.
+# Similarly OCaml has .ml (implementation) and .mli (interface files) and
+# one often needs to switch between them.
+#
+# This command provides a simple functionality that allows you to accomplish this.
+define-command ocaml-ml-switch-mli -docstring 'Switch between .ml and .mli file or vice versa' %{
+    evaluate-commands %sh{
+        if [ "${kak_buffile##*.}" = 'ml' ]; then
+            printf "edit -existing '%s'\n" "${kak_buffile}i"
+        elif [ "${kak_buffile##*.}" = 'mli' ]; then
+            printf "edit -existing '%s'\n" "${kak_buffile%i}"
```

Comment:
I guess we can protect against weird filenames

```suggestion
            printf "edit -existing -- '%s'\n" "$(printf %s "${kak_buffile%i}" | sed "s/'/''/g")"
```

Reply:
Done and made consistent with `go.kak` (e.g. `-existing` removed, see below)